services/apps/alcs/src/portal/application-document/application-document.controller.ts

```diff
@@ -135,12 +135,11 @@ export class ApplicationDocumentController {
   async delete(@Param('uuid') fileUuid: string, @Req() req) {
     const document = await this.applicationDocumentService.get(fileUuid);
 
-    await this.applicationSubmissionService.verifyAccess(
-      document.applicationUuid,
-      req.user.entity,
-    );
-
-    //TODO: How do we control who can delete which document types?
+    //TODO: How do we know which documents applicant can delete?
```

Comment:
no verifyAccess anymore? Is it because LUP should be able to manage these?

Reply:
We can't call verify access as we don't have the file number, just the documents uuid. Might need to think about how we handle this and add fileNumber to the endpoints just so we have it.